GirafIntegrationTest/test.py

```diff
@@ -1,13 +1,31 @@
 import unittest
+import sys
 from requests import get
```

Comment:
Running the tests in Python 3 yields the error seen on the image.

![image](https://user-images.githubusercontent.com/32843490/79799164-54ba1d00-835a-11ea-86e2-3b0362f56f71.png)

Reply:
I may have forgotten to add that the tests rely upon a package called requests. it can be found here: https://requests.readthedocs.io/en/master/. i will edit the main text to reflect this.